app/src/main/java/ch/deletescape/lawnchair/shortcuts/ShortcutsItemView.java

```diff
@@ -110,7 +110,7 @@ public boolean onLongClick(View v) {
                 getContainer(), sv.getFinalInfo(),
                 new ShortcutDragPreviewProvider(sv.getIconView(), mIconShift), new DragOptions());
         dv.animateShift(-mIconShift.x, -mIconShift.y);
-
+        mLauncher.closeFolder(true);
```

Comment:
Fix whitespace please.

Reply:
There isn't

Reply:
There isn't, and code is aligned as well

Reply:
![image](https://user-images.githubusercontent.com/28547847/32984601-0f25a08e-ccaa-11e7-8f8c-fed4602aca09.png)
There is indeed as GitHub shows to me.